test/e2e/specs/nux.test.js

```diff
@@ -103,12 +109,21 @@ describe( 'New User Experience (NUX)', () => {
 		let areTipsEnabled = await getTipsEnabled( page );
 		expect( areTipsEnabled ).toEqual( false );
 
-		// Click again to re-enable tips; they should appear.
-		await clickOnMoreMenuItem( 'Show Tips' );
+		// Open the Options modal.
+		await clickOnMoreMenuItem( 'Options' );
```

Comment:
It's possible to generalize `togglePrePublishChecks` utility method to take the text name as param and put it inside the following string `//label[contains(text(), "Enable Tips")]` and thus simplify this code here and a few lines earlier.

Reply:
How's this? 1a4394918